### DISCUSSION TITLE
Can we add LookAhead optimizer to mmengine?

### DISCUSSION THREAD

**Discussion by twmht** [open-mmlab/mmengine#663] [Q&A] [answered]
LookAhead optimizer works like a optimizer wrapper(https://github.com/rwightman/pytorch-image-models/blob/a6e8598aaf90261402f3e9e9a3f12eac81356e9d/timm/optim/lookahead.py#L20).

How Can we add LookAhead  to mmengine?

**Comment by ZwwWayne** [comment, marked as answer]
We can simply copy the implementation of LookAhead into the optimizer folder and add it into the registry of optimizer.

**Comment by ZwwWayne** [comment]
Would you like to do that?
